### DISCUSSION TITLE
https://angular.love/cracking-angular-di-the-hidden-layers-of-injectors

### DISCUSSION THREAD

**Discussion by giscus** [HouseOfAngular/angular-love#363] [General]
# https://angular.love/cracking-angular-di-the-hidden-layers-of-injectors

Angular.love - a place for all Angular enthusiasts created to inspire and educate.

https://angular.love/cracking-angular-di-the-hidden-layers-of-injectors

**Comment by davayd** [comment]
Thanks for article!

Do you plan add explanation about `inject()` function in different contexts ?

**Reply by ItNoN** [reply]
Hello. Thank you for your comment! 

It is actually a good idea. I hope I will publish it in the future.

**Comment by ngiakhanh96** [comment]
Hi what if we have nested lazy route in module based/standalone/mixed app ? 
Then what does the DI hierarchy look like ? 
Thanks

**Reply by ngiakhanh96** [reply]
I mean nested lazy loading module in lazy loading module
